apm-sniffer/apm-agent-core/src/main/java/org/skywalking/apm/agent/core/context/TracingContext.java

```diff
@@ -98,10 +98,12 @@ public void inject(ContextCarrier carrier) {
             TraceSegmentRef ref = refs.get(0);
             operationId = ref.getEntryOperationId();
             operationName = ref.getEntryOperationName();
+            carrier.setEntryApplicationInstanceId(ref.getEntryApplicationInstanceId());
```

Comment:
Please follow the same coding style like this:
1. assign to a local variable first
1. call `carrier#setEntryApplicationInstanceId`